src/components/Editor/EditorMenu.js

```diff
@@ -107,7 +113,7 @@ function getMenuItems(
     id: "node-menu-show-source",
     label: revealInTreeLabel,
     accesskey: revealInTreeKey,
-    disabled: false,
+    disabled: isOriginal,
```

Comment:
pretty printed files are not in the tree, but other original files that come from a bundle are.

Reply:
Correct. I will fix this.